lib/signaling/v2/peerconnection.js

```diff
@@ -904,6 +913,14 @@ class PeerConnectionV2 extends StateMachine {
     }).then(offer => {
       if (!isFirefox) {
         offer = workaroundIssue8329(offer);
+      } else {
+        // NOTE(mmalavalli): We work around Chromium bug 1106157 by disabling
+        // RTX in Firefox 79+. For more details about the bug, please go here:
+        // https://bugs.chromium.org/p/chromium/issues/detail?id=1106157
+        offer = new this._RTCSessionDescription({
+          sdp: disableRtx(offer.sdp),
```

Comment:
I would also wrap these changes around in a function like:

``` 
offer = workaround 1106157(offer);
```

I am hoping that this bug gets fixed and we can remove this workaround asap.

Reply:
@makarandp0 I would like to make the workaround as transparent as possible so that a casual reader knows what the workaround does, in this case disabling RTX for Firefox SDPs.